src/bionics.cpp

```diff
@@ -430,7 +432,16 @@ bool Character::activate_bionic( int b, bool eff_only )
             }
         }
 
-        const size_t win_h = std::min( static_cast<size_t>( TERMY ), bad.size() + good.size() + 2 );
+        const auto &vit_all = vitamin::all();
+        static const std::set<vitamin_id> trackable_vits = {vitamin_id( "iron" ), vitamin_id( "calcium" ), vitamin_id( "vitA" ), vitamin_id( "vitB" ), vitamin_id( "vitC" ), vitamin_id( "mutant_toxin" )};
+        for( const auto &v : vit_all ) {
+            if( trackable_vits.count( v.first ) ) {
+                vit.push_back( v.second.name() + _( " level is " ) + to_string( vitamin_get( v.first ) ) );
```

Comment:
Just printing the raw integer value here isn't helpful, because nobody knows what that means.
I'd suggest adding a `vitamin::get_string_level()` or something of the sort that gives you a string based on how much of the vitamin you have, and how close you are to each stage of either excess/deficiency.

Reply:
Yeap, that's why I asked for a comment how it should be finally represented. And what those numbers are really mean in reality.